spec/i18n_spec.rb

```diff
@@ -64,6 +80,11 @@
     end
   end
 
+  def extract_interpolation_arguments(translation)
+    matches = translation.scan(I18n::INTERPOLATION_PATTERN).map(&:compact).map(&:first)
```

Comment:
OMG I wish I had known about this regex

Also we can do `Array#to_set` if that's easier than `Set.new(array)` but not a big deal

Reply:
:+1: on `#to_set`. I spend too much time writing javascript and forget these things exist.